libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Properties.hs

```diff
@@ -956,19 +157,22 @@ testTxValidForLEDGER proof (Box _ (trc@(TRC (_, ledgerState, vtx))) _) =
 -- test individual functions in this file.
 
 -- | Construct a random (Gen b)
-makeGen :: Proof era -> (Proof era -> GenRS era b) -> Gen b
-makeGen proof computeWith = do
-  env <- genGenEnv proof
-  (ans, _state, _written) <- runRWST (computeWith proof) env emptyGenState
-  pure ans
+makeGen :: Reflect era => Proof era -> (Proof era -> GenRS era b) -> Gen b
+makeGen proof computeWith = fst <$> runGenRS proof def (computeWith proof)
 
-runTest :: PrettyA a => (Proof era -> GenRS era a) -> Proof era -> IO ()
-runTest computeWith proof = do
+runTest :: (Reflect era, PrettyC a era) => (Proof era -> GenRS era a) -> (a -> IO ()) -> Proof era -> IO ()
```

Comment:
It would be good to put these helper functions somewhere at the bottom with a short that they are intended for development and debugging

Reply:
done